index.js

```diff
@@ -282,6 +283,7 @@ const packages = {
   'vue-admin': 'https://opencollective.com/vue-admin',
   'vue-js-modal': 'https://opencollective.com/vue-js-modal',
   'Vulcan': 'https://opencollective.com/telescope',
+  'vuetify': 'https://www.patreon.com/vuetify',
```

Comment:
Remove this line, there's no need to mention packages explicitly except for team projects. By adding yourself to the `authors` section, your donate link will show up for all the packages you maintain.